charts/redpanda/ci/30-additional-flags-override-novalues.yaml

```diff
@@ -0,0 +1,4 @@
+statefulset:
```

Comment:
Could you add https://github.com/redpanda-data/helm-charts/blob/b14967ada018fc9be743665040a6191f187ed65a/.github/check-ci-files.sh#L14

`novalues.yaml` to the if statement? CI is failing at this moment with

```
verifying file: charts/redpanda/ci/30-additional-flags-override-novalues.yaml
- file does not match neither suffix '-values.yaml' nor -values.yaml.tpl'
```

https://github.com/redpanda-data/helm-charts/actions/runs/8602844060/job/23573322602?pr=1151#step:5:321